feature-utils/poly-look/src/react-components/cards/cardList.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { INITIAL_HISTORY_STATE } from "../../constants/state";
+
+import "./cardList.css";
+
+/**
+ * A list of Cards
+ *
+ * @param {jsx} children Jsx children (Cards)
+ * @param {String} className CSS classes added to the main div
+ * @returns jsx
+ */
+export const CardList = ({ children, className }) => {
+  return <div className={`card-list ${className}`}>{children}</div>;
+};
+
+/**
+ * Card component that makes up a CardList
+ *
+ * @param {jsx} children HTML elements displayed inside the card
+ * @param {Object} navigation Navigation object, if passed will show a button and will trigger actions
+ * @param {History} navigation.history a useHistory generated history (also requires route)
+ * @param {String} navigation.route a router route (also requires history)
+ * @param {Object} navigation.stateChange a router history stateChange (also requires history)
+ * @param {Callback} navigation.onClick onClick function
+ * @param {String} navigation.buttonText the buttons displayed text
+ * @returns jsx
+ */
+export const Card = ({ children, navigation }) => {
+  if (!navigation) return <div className="card">{children}</div>;
+  const { history, route, stateChange, onClick, buttonText } = navigation;
+  if (!buttonText) {
+    console.error("Card: DetailsButton must have text");
+    return null;
+  }
+  if (!(history && route) && !onClick) {
```

Comment:
And then there are two types of NavigableCards: ClickableCards or RoutedCards? Does that sound about right?

Reply:
`RoutableCards`, whatever.

Reply:
I mean they route `onClick`